backend/internal/application/usecase/hello_world_usecase_test.go

```diff
@@ -0,0 +1,119 @@
+package usecase
+
+import (
+	"context"
+	"errors"
+	"myapp/internal/domain/model"
+	"myapp/internal/domain/repository"
+	"myapp/internal/domain/repository/repository_mock"
+	"reflect"
+	"testing"
+
+	"go.uber.org/mock/gomock"
+)
+
+func TestNewHelloWorldUsecase(t *testing.T) {
+	type args struct {
+		r repository.HelloWorldRepository
+	}
+	tests := []struct {
+		name string
+		args args
+		want HelloWorldUsecase
+	}{
+		{
+			name: "success",
+			args: args{
+				r: &repository_mock.MockHelloWorldRepository{},
+			},
+			want: HelloWorldUsecase{
+				r: &repository_mock.MockHelloWorldRepository{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := NewHelloWorldUsecase(tt.args.r); !reflect.DeepEqual(got, tt.want) {
```

Comment:
gotだと情報量すくないから、変数名はuとかの方がいいかも

そしてassertパッケージを使うとif文要らなくなるから、36行目はこうしたい！

```suggestion
			u := NewHelloWorldUsecase(tt.args.r);
```

Reply:
[415e5c4](https://github.com/givery-bootcamp/dena-2024-team3/pull/87/commits/415e5c4084cae6db3748cdc77592e59eec70e5e6)で修正した！